go-apps/meep-sandbox-ctrl/server/sandbox-ctrl.go

```diff
@@ -767,11 +772,16 @@ func sendEventScenarioUpdate(event dataModel.Event) (error, int, string) {
 // Retrieve element name from type-specific structure
 func getScenarioNodeName(node *dataModel.ScenarioNode) string {
 	name := ""
-	if node.Type_ == mod.NodeTypeUE {
+	if mod.IsPhyLoc(node.Type_) {
```

Comment:
`if node.Type_ == mod.NodeTypeUE {` became `if mod.IsPhyLoc(node.Type_) {`
This change does not seem equivalent to me as a UE is a Physical Location but not all Physical locations are UEs.
Please clarify...

Reply:
The previous implementation supported only UE type. Now the API supports all physical location and process types. This function was updated to check for all physical location and process types.